Gaijinizer patch/Map037.txt

```diff
@@ -1117,44 +1145,48 @@ I am at this ...#MTLed
 結果はどうであれ、\\N[1]様には
 沢山『触れる』ことができましたから……
 > CONTEXT: Map037/events/4/pages/1/524/Dialogue/yuuma
-It was a fun time. Whatever the outcome, for
-\\N[1]. Because I was able to "touch" a lot ...#MTLed
+I enjoyed our brief time together.
+
+Whatever happens, I was able to "touch"
+\\N[1]-sama a lot...#Ron
 > END STRING
 
 > BEGIN STRING
 それでは、ごきげんよう。
 
 貴方様のご武運を、御祈りしております………
 > CONTEXT: Map037/events/4/pages/1/529/Dialogue/yuuma
-Well then. I pray for your good luck .........#MTLed
+Well then, good day to you.
+
+I will pray for your good fortune...#Ron
 > END STRING
 
 > BEGIN STRING
 アイアンメイデンはサラサラと、
 
 まるで風にさらわれる灰のように消滅した………
 > CONTEXT: Map037/events/4/pages/1/542/Dialogue
-Iron Maiden is smoothy, It disappeared like ash that was
-swept away by the wind .........#MTLed
+Iron Maiden disappeared like wind blowing away
+ash...#Ron
 > END STRING
 
 > BEGIN STRING
 ……こいつも、俺たちと同じで
 
 Ｍ.Ｏ.Ｗに翻弄されていただけなのかもな。
 > CONTEXT: Map037/events/4/pages/1/548/Dialogue/yuuma
-...... This is always the same as us
-M.O.W. may have been taunted.#MTLed
+... Just like us, she might have also had her life
+toyed with by M.O.W.#Ron
 > END STRING
 
 > BEGIN STRING
 理由はどうであれ、奴らに身体を変化させられて
 
 失敗すれば消されると分かっていながら、お前を見逃した。
 > CONTEXT: Map037/events/4/pages/1/552/Dialogue/yuuma
-Whatever the reason, they can change their body
-I missed you while I knew it would be erased if I
-failed.#MTLed
+They can be forced to change their form for
+whatever reason. Even though she knew they'd make
+her disappear if she failed, she still helped you.#Ron
 > END STRING
```

Comment:
I like the original intention of "erased" as opposed to "disappear". Disappear sounds a lot softer and less harsh, and I think this goes more for the shock value of the whole experience. I'd phrase it like this:
>_**Even though she knew she'd be erased if she failed, she still helped you.**_